packages/terra-base/src/clinical-lowlight-theme/Base.scss

```diff
@@ -0,0 +1,18 @@
+:local {
```

Comment:
make this global

Reply:
Updated here https://github.com/cerner/terra-core/pull/3015/commits/491eca803d32a8d280a259dfdac22897a2dccfbe.